core/src/main/java/tc/oc/pgm/core/CoreMatchModule.java

```diff
@@ -155,10 +155,7 @@ public void lavaProtection(final BlockTransformEvent event) {
   public void onObjectiveModeSwitch(final ObjectiveModeChangeEvent event) {
     for (Core core : this.cores) {
       if (core.isAffectedByModeChanges()) {
```

Comment:
You no longer need isAffectedByModeChanges.

If the list of modes is an empty collection, that's your validation that this is in fact, not affected by modes